### DISCUSSION TITLE
Connection name

### DISCUSSION THREAD

**Discussion by obonilla66** [dbeaver/dbeaver#19593] [Ideas]
When I create a new connection, you chose to select the database driver first (it is OK), but the second most important thing is the name with which one will find that connection. 
Please consider requesting the name of the connection where the server data are entered. This is very important, especially when you have defined a lot of connections to many different servers, or many different schemas o databases in the same server.

Thank you.

**Comment by LonwoLonwo** [comment]
Hello. Changing the connection pattern may help.

![2023-04-04 10_48_49-DBeaver Team Edition 23 0 2](https://user-images.githubusercontent.com/45152336/229724477-3c442235-fcb9-445c-abd6-b84851436c50.png)

**Comment by Robin-Britz** [comment]
You can give your newly added connection an alias after you've added it.
Right click connection > Edit connection > General > Connection Name > OK

![image](https://github.com/dbeaver/dbeaver/assets/36510262/7c574b7c-6c5b-4559-9e9d-2940e4a19bd9)
